packages/kit/src/packaging/index.js

```diff
@@ -86,6 +90,9 @@ export async function build(config, cwd = process.cwd()) {
  * @param {import('types').ValidatedConfig} config
  */
 export async function watch(config, cwd = process.cwd()) {
+	// Make sure generated tsconfig is up-to-date
+	write_tsconfig(config);
+
```

Comment:
you were also adding this to the `build` command which is called on the next line, so we probably don't need to do it twice?
```suggestion
```

Reply:
Yep!